### ISSUE TITLE
ppn2ocr: FULLTEXT should reference images

### ISSUE THREAD

**Issue by mikegerber** [mikegerber/my_ocrd_workflow#16]
We assume that FULLTEXT ALTO was created using the BEST images. It should also reference these images.
                                                                                                   
  * Insert or check `<imageFilename>`                                                                          
  * Sanity check image:                                                                                                 
    * image dimensions against `<Page HEIGHT= WIDTH= />`                                                                
      * There may be documents where this is not the case. Which ones?                                                  
    * also check that `<MeasurementUnit>pixel</MeasurementUnit>`

**Comment by cneud** [comment]
For our own digitised collections, I strongly support appropriate sanity checks - but also we should check this with colleagues ;)

E.g. I know that in the past the fulltext production has at times included the rescaling of TIFF images after OCR (and then according transformation of coordinates in fulltext) and also the transformation from ABBYY XML to ALTO.

**Comment by mikegerber** [comment]
> For our own digitised collections, I strongly support appropriate sanity checks - but also we should check this with colleagues ;)

Yep, that I have on the radar. Especially because we do some guessing here, e.g. we deduce the IIIF URLs from the `/dms/` URLs or assume (as stated above) that the "full" images are exactly those that were used for FULLTEXT. 

→ Sanity check this
→ Communicate internally and see if anything can be improved elsewhere (e.g. IIIF URLs with the right resolution in the METS in the first place)
